Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# 1st localisation crusade

**And so it begins again**

## Changelog:
- Renaming some titles (german)
- Working on traits file (german)
- Finished localization in the religions file (german)
- Fixed a small bug with the german localization of MaA
- Finished translation of the core tenets file (german)
- Finished translation of the magical doctrines file (german)
- Translated changes made to the English localization (german)

- Changed one cultural adjective in the replacement file (English)
- Localized the Religion game concept description (game_concept_religion_desc)
- Localized the Religious Family game concept description (game_concept_religious_family_desc)
- Localized the Knight game concept description (game_concept_knight_desc)
- Localized heresy pop up game setting (setting_default_regional_heresy_desc)
- Localized the Organized game concept description (game_concept_organized_desc)
- Localized innovations (innovation_development_03_desc & innovation_cranes_desc)
- Localized Dynasty, Last unlock in the Blood Legacy tree (blood_legacy_5_name)
- Localized Praetorian Guard and Parthian Tactics perk names
- Many changes based on Issue 101: https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/issues/101
- Literally this for all the words that are different between the two dialects: 
![image](https://user-images.githubusercontent.com/74790101/111005731-55381980-8359-11eb-88c5-8c6edcbcdb52.png)




## How to test:
German:
- Check the different religions, tenets and doctrines in-game. You'd have to switch faiths or persons to see all of the doctrines because for example shaman or goblin tenets tend not to appear when you're of another faith group.
- Check the changes that got made to the English localization, too.
- (You can simply only read the changes in the file, too, if that's easier for you, but when new to ck3 localization code, it might help to check how it plays out in-game.)

English:
- Check the changes that got made to the English localization.
- (You can simply only read the changes in the file, too, if that's easier for you, but when new to ck3 localization code, it might help to check how it plays out in-game.)
